src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ConfirmationService.kt

```diff
@@ -107,19 +107,15 @@ class ConfirmationService(
     return result.toResponse()
   }
 
-  fun Confirmation.toEvent(inmateDetail: InmateDetail, type: ArrivalType) = ArrivalEvent(
+  private fun Confirmation.toEvent(inmateDetail: InmateDetail, type: ArrivalType) = ArrivalEvent(
     movementId = if (this is Confirmation.Expected) this.arrivalId else null,
-    prisonId = this.detail.prisonId!!,
+    prisonId = inmateDetail.agencyId,
```

Comment:
Just checking this is correct? If agencyId is being used in place of prisonId, would it be preferable to rename the variable to agencyId?

Reply:
It does refer to the same thing - just innaccurately named in prison API.

An agency refers to prisons, courts, police stations, etc... in this case it will always be the ID associated with a prison. 

The service exposes the "more correct" name - we could have mapped it using jackson annotations

Reply:
Thanks for clarifying, I was unaware of the Prison API inaccurately. Good to know!